examples/mifare-desfire-format.c

```diff
@@ -111,6 +111,10 @@ main(int argc, char *argv[])
 	    if (format) {
 		int res;
 
+		// Sometimes format takes a bit longer
```

Comment:
This feels wrong (see bellow).  I think that it's better to not have some API that do not work with default settings, and in this case, this file should not be modified.

Reply:
I intentionally added a usage example of freefare_set_tag_timeout in the example code.
mifare-desfire-format was the only logic place IMHO

Reply:
If the value defaults to 0 (disable timeout), it may make sense to update all examples to set a custom timeout.  What does not make sense is that the `mifare_desfire_format()` fails if you call it with the default timeout :wink: